server/src/db/migrations/1675360350104-roles_table.ts

```diff
@@ -4,11 +4,11 @@ export class rolesTable1675360350104 implements MigrationInterface {
     name = 'rolesTable1675360350104'
 
     public async up(queryRunner: QueryRunner): Promise<void> {
-        await queryRunner.query(`CREATE TABLE "role" ("id" SERIAL NOT NULL, "name" character varying NOT NULL, CONSTRAINT "PK_b36bcfe02fc8de3c57a8b2391c2" PRIMARY KEY ("id"))`);
+        await queryRunner.query(`CREATE TABLE "roles" ("id" SERIAL NOT NULL, "name" character varying NOT NULL, CONSTRAINT "PK_b36bcfe02fc8de3c57a8b2391c2" PRIMARY KEY ("id"))`);
     }
 
     public async down(queryRunner: QueryRunner): Promise<void> {
-        await queryRunner.query(`DROP TABLE "role"`);
+        await queryRunner.query(`DROP TABLE "roles"`);
```

Comment:
We can not change tables names in migrations because we won't be able to roll back
Take it back as it was, I will handle that